src/Tribe/Customizer.php

```diff
@@ -115,11 +115,16 @@ public function __construct() {
 		add_action( 'wp_print_footer_scripts', [ $this, 'print_css_template' ], 15 );
 
 		// front end styles from customizer
-		add_action( 'wp_enqueue_scripts', [ $this, 'inline_style' ], 15 );
 		add_action( 'tribe_events_pro_widget_render', [ $this, 'inline_style' ], 101 );
 		add_action( 'wp_print_footer_scripts', [ $this, 'shortcode_inline_style' ], 5 );
 		add_action( 'wp_print_footer_scripts', [ $this, 'widget_inline_style' ], 5 );
 
+		// Get print styles action based on whether v2 events or v2 tickets exist and are enabled.
+		$is_events_v2        = function_exists( 'tribe_events_views_v2_is_enabled' ) && tribe_events_views_v2_is_enabled();
+		$is_tickets_v2       = function_exists( 'tribe_events_tickets_views_v2_is_enabled' ) && tribe_events_tickets_views_v2_is_enabled();
+		$print_styles_action = $is_events_v2 || $is_tickets_v2 ? 'wp_print_footer_scripts' : 'wp_enqueue_scripts';
+		add_action( $print_styles_action, [ $this, 'inline_style' ], 15 );
```

Comment:
Would it be better to have TEC and ET filter the print styles action to manually override it to print footer scripts instead of putting the logic direct in Common?